scrapy/exporters.py

```diff
@@ -144,8 +144,10 @@ def _export_xml_field(self, name, serialized_value):
         elif is_listlike(serialized_value):
             for value in serialized_value:
                 self._export_xml_field('value', value)
-        else:
+        elif isinstance(serialized_value, (six.text_type, bytes)):
```

Comment:
shouldn't this be testing only against `six.text_type`?
we're not expecting bytes to get here at this point, are we?